src/odemis/gui/comp/miccanvas.py

```diff
@@ -1264,6 +1270,20 @@ def setView(self, view, tab_data):
         self.view = view
         self._tab_data_model = tab_data
 
+        # handle cross hair
+        self.view.show_crosshair.subscribe(self._on_cross_hair_show, init=True)
```

Comment:
This is going to show the crosshair on all the TwoDPlotCanvas. So all the one in the analysis (both line spectrum and temporal spectrum), and in acquisition tab (temporal spectrum). I'm pretty sure that in analysis it should be hidden by default. In the acquisition tab, I let you decide.
To hide the crosshair by default, I'd suggest to either autmatically set the VA to `False` in the viewport.set_view(), or at the view creation (in the tab controller).